packages/@lwc/engine/src/framework/main.ts

```diff
@@ -10,7 +10,7 @@ import '../polyfills/proxy-concat/main';
 import '../polyfills/aria-properties/main';
 
 // TODO [#1296]: Revisit these exports and figure out a better separation
-export { createElement } from './upgrade';
+export { createElement } from './create-element';
```

Comment:
if you rename this, it is going to be crazy to upgrade/rebase this branch, we can change that name later on